lib/tmdb_handler.rb

```diff
@@ -9,196 +9,6 @@ def initialize(message)
-      profile: MoviePersonProfile.parse_result(bio_results),
-      movie_credits: MoviePersonCredits.parse_result(movie_credits_results),
-      tv_credits: TVPersonCredits.parse_result(tv_credits_results)
-    )
-  end
-
-  def tmdb_handler_actor_credit(credit_id)
-    credit_url = "#{BASE_URL}/credit/#{credit_id}?api_key=#{ENV['tmdb_api_key']}"
-    credit_data = JSON.parse(open(credit_url).read, symbolize_names: true)
-    TVActorCredit.parse_record(credit_data)
-  end
-
-  def tmdb_handler_tv_series_search(query)
-    search_url = "#{BASE_URL}/search/tv?query=#{query}&api_key=#{ENV['tmdb_api_key']}"
-    tmdb_response = JSON.parse(open(search_url).read, symbolize_names: true)
-    discover_results = tmdb_response[:results]
-    TVSeries.parse_search_records(discover_results) if discover_results.present?
-  end
-
-  def tmdb_handler_tv_series_autocomplete(query)
-    search_url = "#{BASE_URL}/search/tv?query=#{query}&api_key=#{ENV['tmdb_api_key']}"
-    tmdb_response = JSON.parse(open(search_url).read, symbolize_names: true)
-    tmdb_response[:results].map{ |result| result[:name] }.uniq
-  end
-
-  def tmdb_handler_tv_series(show_id)
-    show_url = "#{BASE_URL}/tv/#{show_id}?api_key=#{ENV['tmdb_api_key']}&append_to_response=credits"
-    series_data = JSON.parse(open(show_url).read, symbolize_names: true)
-    TVSeries.parse_record(series_data, show_id)
-  end
-
-  def tmdb_handler_tv_season(series:, show_id:, season_number:)
-    season_url = "#{BASE_URL}/tv/#{show_id}/season/#{season_number}?api_key=#{ENV['tmdb_api_key']}&append_to_response=credits"
-    season_results = JSON.parse(open(season_url).read, symbolize_names: true)
-    TVSeason.parse_record(
-      series: series,
-      show_id: show_id,
-      season_data: season_results
-    )
-  end
-
-  def tmdb_handler_tv_episode(show_id:, season_number:, episode_number:)
-    episode_url = "#{BASE_URL}/tv/#{show_id}/season/#{season_number}/episode/#{episode_number}?api_key=#{ENV['tmdb_api_key']}"
-    episode_data = JSON.parse(open(episode_url).read, symbolize_names: true)
-    TVEpisode.parse_record(episode_data)
-  end
-
-  def tmdb_handler_search_common_actors_in_two_movies(movie_one_title, movie_two_title)
-    movie_one_results = tmdb_handler_search(movie_one_title)
-    movie_two_results = tmdb_handler_search(movie_two_title)
-    not_found_message = movie_one_results.not_found_message.presence || movie_two_results.not_found_message.presence
-
-    if not_found_message.present?
-      OpenStruct.new(not_found_message: not_found_message)
-    else
-      movie_one = tmdb_handler_movie_more(movie_one_results.movies.first.tmdb_id)
-      movie_two = tmdb_handler_movie_more(movie_two_results.movies.first.tmdb_id)
-      OpenStruct.new(
-        movie_one: movie_one,
-        movie_two: movie_two,
-        common_actors: movie_one.actors & movie_two.actors,
-        not_found_message: nil
-      )
-    end
-  end
-
   def tmdb_handler_discover_search(params)
```

Comment:
This last method has more refactoring work to do. I started that work in PR #280.